core/src/main/java/org/elasticsearch/common/geo/builders/PolygonBuilder.java

```diff
@@ -141,9 +142,10 @@ protected void validateHole(LineStringBuilder shell, LineStringBuilder hole) {
 
         Edge[] edges = new Edge[numEdges];
         Edge[] holeComponents = new Edge[holes.size()];
-        int offset = createEdges(0, orientation, shell, null, edges, 0);
+        final AtomicBoolean translate = new AtomicBoolean(false);
```

Comment:
++ you're right. Without reverting back to a member variable or unnecessarily implementing something I don't believe there is any thing else.